lib/Github/Api/Organization/Teams.php

```diff
@@ -32,12 +32,22 @@ public function create($organization, array $params)
         return $this->post('/orgs/'.rawurlencode($organization).'/teams', $params);
     }
 
-    public function show($team)
+    /**
+     * @link https://developer.github.com/v3/teams/#list-teams
+     */
+    public function show($team, $organization = null)
     {
+        if ($organization) {
+            return $this->get('/orgs/'.rawurlencode($organization).'/teams/'.rawurlencode($team));
+        }
+
```

Comment:
```suggestion
    if (null === $organization) {
        return $this->get('/teams/'.rawurlencode($team));
    }

    return $this->get('/orgs/'.rawurlencode($organization).'/teams/'.rawurlencode($team));
```

I would actually turn the if around and have case for the old behaviour. (this is also the case for the other methods here)
Also I would add a `@trigger_error('Not passing the $organisation parameter is deprecated in knpLabs/php-github-api v2.14 and will be mandatory in v3.0.', E_USER_DEPRECATED);` in the if so users (who optin to the deprecation system will get a warning)